### PR TITLE
Mkirk/fixup bubble shape

### PR DESCRIPTION
**Pull request by michaelkirk-signal**
tweaking our corner style, radius, height to more closely match the spec.

### CODE REVIEW

File: Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleView.m [hunk 1]
```diff
@@ -161,19 +161,52 @@ + (UIBezierPath *)maskPathForSize:(CGSize)size
     CGFloat bottomRounding
         = (useSmallCorners_Bottom ? kOWSMessageCellCornerRadius_Small : kOWSMessageCellCornerRadius_Large);
 
+    const CGFloat topAngle = 3.0f * M_PI / 2.0f;
+    const CGFloat rightAngle = 0.0f;
+    const CGFloat bottomAngle = M_PI / 2.0f;
+    const CGFloat leftAngle = M_PI;
```
Comment (charlesmchen-signal) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleView.m]: This is fine, but FYI CG has a bunch of pre-existing but confusingly-named constants like `M_PI_2` for these purposes.

File: Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleView.m [hunk 2]
```diff
@@ -161,19 +161,52 @@ + (UIBezierPath *)maskPathForSize:(CGSize)size
...
+
     [bezierPath moveToPoint:CGPointMake(bubbleLeft + topRounding, bubbleTop)];
+
+    // top line
     [bezierPath addLineToPoint:CGPointMake(bubbleRight - topRounding, bubbleTop)];
```
Comment (charlesmchen-signal) [inline Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleView.m]: I wonder if the `addLineToPoint` calls are now redundant, e.g. whether `UIBezierPath` will "close" the shape between the arcs.

Comment (michaelkirk-signal) [reply Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleView.m]: Hmm.. it appears that it does. I've made this change.

File: Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleView.m (hunk 1 above)
Comment (michaelkirk-signal) [reply Signal/src/ViewControllers/ConversationView/Cells/OWSBubbleView.m]: updated, thanks!

**Pull request merged**